common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -5106,25 +5106,25 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
       "Specifies the minimum amount of tasks, executed by a particular LLAP daemon, before the health\n" +
       "status of the node is examined."),
     LLAP_NODEHEALTHCHECKS_MININTERVALDURATION(
-      "hive.llap.nodehealthckecks.minintervalduration", "300s",
+      "hive.llap.nodehealthchecks.minintervalduration", "300s",
```

Comment:
actually; this is a breaking change - but I don't think anyone have *ever* changed these configs....so lets keep them